Packs/Core/Playbooks/playbook-Exchange_forwarding_rule_configured_README.md

```diff
@@ -0,0 +1,81 @@
+This playbook addresses the following alerts:
+
+- External Exchange inbox forwarding rule configured.
+- Suspicious Exchange inbox forwarding rule configured.
+- Suspicious Exchange email-hiding inbox rule.
+
+Playbook Stages:
+ 
+Triage: 
+
+- The playbook retrieves the caller's IP, the forwarding email address, and the domain.
+
+Early Containment:
+
+- The playbook checks if the IP or domain of the forwarding email address is malicious. If so, it suggests blocking the IP using PAN-OS while continuing the investigation in parallel.
+
+Investigation:
+
+- The playbook checks for suspicious behaviors, including whether an Exchange admin created the rule outside of working hours, from unusual geolocation, or if the user who created the rule has a high-risk score. It then aggregates all evidence collected during the investigation.
+
+Containment:
+
+- If at least two suspicious evidence are found, the playbook executes soft response actions, including signing the user out and deleting the forwarding email address from the user account mailbox. The user will be notified of these actions via email.
```

Comment:
Change 
-  If at least two suspicious evidence are found, the playbook executes soft response actions, including signing the user out and deleting the forwarding email address from the user account mailbox. The user will be notified of these actions via email.
- If two or more pieces of suspicious evidence are found, the playbook will initiate soft response actions. These include signing the user out of their account and removing the forwarding email address from their mailbox. The user will be notified of these actions via email.

Reply:
This descriptions does not mention the hard remediation and the action we take there. please fix